sdk/servicebus/Azure.Messaging.ServiceBus/api/Azure.Messaging.ServiceBus.netstandard2.0.cs

```diff
@@ -425,6 +425,36 @@ public enum ServiceBusTransportType
         AmqpWebSockets = 1,
     }
 }
+namespace Azure.Messaging.ServiceBus.Amqp
+{
+    public enum AmqpBodyType
+    {
+        Unspecified = 0,
+        Data = 1,
+        Sequence = 2,
+        Value = 3,
+    }
+    public static partial class ServiceBusMessageExtensions
+    {
+        public static Azure.Messaging.ServiceBus.Amqp.AmqpBodyType GetAmqpBodyType(this Azure.Messaging.ServiceBus.ServiceBusMessage message) { throw null; }
+        public static System.Collections.Generic.IEnumerable<System.ReadOnlyMemory<byte>> GetAmqpDataBody(this Azure.Messaging.ServiceBus.ServiceBusMessage message) { throw null; }
```

Comment:
I think these should be `IList` rather than `IEnumerable` as they are output. 
https://azure.github.io/azure-sdk/dotnet_introduction.html#dotnet-service-return-model-collections

Reply:
Okie dokie. Will change.

Reply:
Will this require an allocation (so we can support the indexer) that we would not have to do if we returned IEnumerable?

Reply:
We are already allocating in our implementation, so it shouldn't be any more overhead.